LDAP-Auth/Config/PluginConfiguration.cs

```diff
@@ -101,6 +102,16 @@ public PluginConfiguration()
         /// </summary>
         public bool CreateUsersFromLdap { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow password change.
+        /// </summary>
+        public bool AllowPassChange { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to use Active Directory specific configurations.
+        /// </summary>
+        public bool ActiveDirectory { get; set; }
```

Comment:
Instead of adding an Active Directory specific config flag, the password field should be editable

Reply:
Okay, my plan was to allow an easy config of AD but that would definitely be a seperate pull at least.  I will pull all those related pulls (lol) and get them into a new pull.